docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -99,6 +99,14 @@ Added a new object that allows you to manage the delivery date and time of a mes
 
 This feature was released in [requirement set 1.10](../requirement-set-1.10/outlook-requirement-set-1.10.md). However, additional events are now available in preview. To learn more, refer to [Supported events](../../../outlook/autolaunch.md#supported-events).
 
+**Available in**: Outlook on Windows (connected to a Microsoft 365 subscription), Outlook on the web (modern)\*, Outlook on Mac (new UI)\*
+
+\* *Some events are only supported on certain clients. For guidance, refer to the [list of events and their supported clients](../../../outlook/autolaunch.md#supported-events).*
```

Comment:
"Available in" should represent where the preview events are available so just noting Windows should suffice. We also link to the supported events in the previous paragraph. Consider updating the text link if anything
```suggestion
**Available in**: Outlook on Windows (connected to a Microsoft 365 subscription)
```